packages/client/playwright/components/spotTile.component.ts

```diff
@@ -24,10 +25,19 @@ export default class SpotTileComponent extends BaseComponent {
     super(page.getByRole("region").locator("div"), page)
   }
 
+  public getToggle() {
+    return this.page.getByTestId("toggleButton")
+  }
+
+  public async getTileSate() {
+    const tileState = await this.page.evaluate(() => {
+      return window.localStorage.getItem("selectedView")
+    })
```

Comment:
I may be wrong but I think you only need to return `window.window.localStorage.getItem("selectedView")`

Reply:
You're right, I've simplified it accordingly to keep it cleaner 👌🏼